lib/nunjucks/views/manage-prototype/password.njk

```diff
@@ -20,8 +20,8 @@
 
 {% block scripts %}
   <!-- We only need Frontend js in order to focus the error summary -->
-  <script src="/plugin-assets/govuk-frontend/govuk/all.js"></script>
-  <script>
+  <script type="module" src="/plugin-assets/govuk-frontend/dist/govuk/all.bundle.js"></script>
```

Comment:
Should this be changed to use the `/manage-prototype/*` router?

Otherwise plugin assets might resolve either GOV.UK Frontend v4 or v5?

Reply:
Instead, I've used new `govukFrontend.*` locals mentioned in https://github.com/alphagov/govuk-prototype-kit/pull/2306#discussion_r1302704311

```html
<script type="module" src="/plugin-assets/{{ govukFrontend.entryPath }}"></script>
```